index.html

```diff
@@ -54,18 +54,18 @@
 
       @keyframes leftToRight {
         from {
-          left: 0%;
+          transform: translate(0%);

```

Comment:
Jednostki w CSS są opcjonalne, gdy wartość reguły to 0. Możesz usunąć % z tej linijki.